addons/binding/org.openhab.binding.cm11a/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="cm11a"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <bridge-type id="cm11a">
+        <label>CM11a Binding for X10 devices</label>
+        <description>CM11 is a device that allows control of X10 devices for a computer. </description>
+
+        <config-description>
+            <parameter name="serialPort" type="text" required="true" >
+                <label>SerialPort</label>
+                <description>Serial port used to communicate with the CM11a
+                </description>
+            </parameter>
+            <parameter name="refresh" type="integer" min="1">
+                <label>Refresh interval</label>
+                <description>Specifies the refresh interval in seconds.</description>
+                <default>60</default>
+            </parameter>
+        </config-description>
+    </bridge-type>
+
+    <thing-type id="dimmer">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="cm11a"/>
+        </supported-bridge-type-refs>
+
+        <label>X10 Dimmer</label>
+        <description>Controls dimmable loads</description>
+
+        <channels>
+            <channel id="lightlevel" typeId="lightDimmer"/>
+        </channels>
+
+        <config-description>
+            <parameter name="HouseUnitCode" type="text" required="true">
+                <label>X10 House and Unit code</label>
+                <description>Address of dimmer in the X10 system</description>
+            </parameter>
+            <parameter name="Location" type="text">
+                <label>Location of the module</label>
+                <description>Where the X10 module is located</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <thing-type id="switch">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="cm11a"/>
+        </supported-bridge-type-refs>
+
+        <label>X10 Switch</label>
+        <description>On/off switch</description>
+
+        <channels>
+            <channel id="switchstatus" typeId="switchState"/>
+        </channels>
+
+        <config-description>
+            <parameter name="HouseUnitCode" type="text" required="true">
+                <label>X10 House and Unit code</label>
+                <description>Address of switch in the X10 system</description>
+            </parameter>
+            <parameter name="Location" type="text">
```

Comment:
This parameter is nowhere mentioned in the documentation and it sounds to me as if it does not really make sense as a configuration parameter. Can it be removed?